### PR TITLE
remove [] formatting

### PR DESCRIPTION
**Pull request by primaeval**
Here is a less controversial PR.
It removes all the [.*?] formatting and sorts the addons by lowercase name.
If you want to make it more precise the regex needs to look for these label codes:
http://kodi.wiki/view/Label_Formatting

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.1.3'
+        classpath 'com.android.tools.build:gradle:2.2.3'
```
Comment (poisdeux) [inline build.gradle]: Unfortunately, 2.2.3 breaks the unit tests. So we stick to 2.1.3 for now, until someone (probably me) puts in some effort to fix this.

**Comment by SyncedSynapse** [comment]
I would merge this, but i'd like to hear @poisdeux opinion, given that he is currently doing a major refactoring of the fragments, and that includes `AddonDetailsFragment`.

What do you want me to do @poisdeux ?

**Comment by poisdeux** [comment]
It's fine if you merge this first. However, it would be nice to get rid of the gradle revert commit before you merge :).

**Comment by primaeval** [comment]
Won't "Squash and merge" should effectively remove it?

**Comment by SyncedSynapse** [comment]
Yes, i'll squash and merge it later.

**Pull request merged**